receiver/dcgmreceiver/client.go

```diff
@@ -110,7 +112,8 @@ func newClient(config *Config, logger *zap.Logger) (*dcgmClient, error) {
 	}, nil
 }
 
-// TODO: b/293585142 - Remove this function and use dcgm.Init() to check if DCGM is
+// isDcgmInstalled returns true if it was able to load the dcgm shared library.
```

Comment:
[Optional] Isn't DCGM an acronym? If so, it should be capitalized in comments here and below.

Reply:
Done!